### PR TITLE
docs: misc changes triggered while revisiting translations (smoe:docs29_mac_104)

### PR DESCRIPTION
**Pull request by smoe**
Not too much left.

### CODE REVIEW

File: docs/src/code/writing-tests.adoc [hunk 1]
```diff
@@ -34,8 +33,8 @@ _test.hal_ below the directories specified on the command line, or under
 `tests/` if no command line argument is specified. These files
 represent three different ways to run the tests.
 
-The _runtests_ script accepts the following arguments, see the output
-from `scripts/runtests -h` for the authoritative list:
+.Arguments acceppted by The _runtests_ script, see the output
```
Comment (hansu) [inline docs/src/code/writing-tests.adoc:36]:
```suggestion
.Arguments accepted by The _runtests_ script, see the output
```

File: docs/src/ladder/classic-ladder.adoc [hunk 2]
```diff
@@ -395,46 +395,46 @@ Represent up/down counters.
...
-* '%Cxx.D' - Counter xx done (Boolean, read only)
-* '%Cxx.E' - Counter xx empty overflow (Boolean, read only)
-* '%Cxx.F' - Counter xx full overflow (Boolean, read only)
-* '%Cxx.V' - Counter xx current value (integer, read or write)
-* '%Cxx.P' - Counter xx preset (integer, read or write)
+* `%C`__xx__`.D` - Counter _xx_ done (Boolean, read only)
+* `%C`__xx__`.E` - Counter _xx_ empty overflow (Boolean, read only)
+* `%C`__xx__`.F` - Counter _xx_ full overflow (Boolean, read only)
+* `%C`__xx__`.V` - Counter _xx_ current value (integer, read or write)
+* `%C`__xx__`.P` - Counter _xx_ preset (integer, read or write)
```
Comment (hansu) [inline docs/src/ladder/classic-ladder.adoc:416]:
```suggestion
* '%C'__xx__`.P` - Counter _xx_ preset (integer, read or write)
```
Please leave them in italic to keep it consistent

File: docs/src/ladder/classic-ladder.adoc [hunk 3]
```diff
@@ -395,46 +395,46 @@ Represent up/down counters.
...
-* 'E' - under flow (output) will be true when the count rolls over from 0 to 9999.
-* 'D' - done (output) will be true when the count equals the preset.
-* 'F' - overflow (output) will be true when the count rolls over from 9999 to 0.
+* `R` - reset (input) will reset the count to 0.
+* `P` - preset (input) will set the count to the preset number assigned from the edit menu.
+* `U` - up count (input) will add one to the count.
+* `D` - down count (input) will subtract one from the count.
+* `E` - under flow (output) will be true when the count rolls over from 0 to 9999.
+* `D` - done (output) will be true when the count equals the preset.
+* `F` - overflow (output) will be true when the count rolls over from 9999 to 0.
```
Comment (hansu) [inline docs/src/ladder/classic-ladder.adoc:404]:
```suggestion
* 'F' - overflow (output) will be true when the count rolls over from 9999 to 0.
```
Please leave them in italic to keep it consistent with the rest

**Review by hansu — CHANGES_REQUESTED** [review]
And please rebase to resolve the conflict.

**Comment by smoe** [comment]
@hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.

**Comment by hansu** [comment]
> @hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.

I haven't done any changes to your branch. But the change from my first comment seems got lost.

**Comment by smoe** [comment]
> > @hansu, I messed up, performed the rebase without first pulling your changes. Please kindly have a look at [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) to ensure that this addresses all in the way you wanted this to be addressed.
> 
> I haven't done any changes to your branch. But the change from my first comment seems got lost.

Those changes from your comments I have meant to have reintroduced in [a1359dd](https://github.com/LinuxCNC/linuxcnc/pull/2592/commits/a1359dddab1fc8cc15fc7f17336d7213a13da452) . I had felt just fine wrt having addressed all issues, just I was not sure if I was interpreting all your changes correctly. What change from your first comment are you referring to?

File: docs/src/code/writing-tests.adoc [hunk 4]
```diff
@@ -34,8 +33,7 @@ _test.hal_ below the directories specified on the command line, or under
 `tests/` if no command line argument is specified. These files
 represent three different ways to run the tests.
 
-The _runtests_ script accepts the following arguments, see the output
-from `scripts/runtests -h` for the authoritative list:
+.Arguments acceppted by The _runtests_ script, see the output from `scripts/runtests -h` for the authoritative list.
```
Comment (hansu) [inline docs/src/code/writing-tests.adoc:36]: This one:

```suggestion
.Arguments accepted by the _runtests_ script, see the output from `scripts/runtests -h` for the authoritative list.
```

**Comment by smoe** [comment]
Accepted your suggested change, and also pulled it into my local repository, just to be sure :)

**Comment by smoe** [comment]
There is no docs29_mac_105 branch on my side, so this may indeed be the last change from my side that would not contribute some content. Happy!

**Comment by hansu** [comment]
> There is no docs29_mac_105 branch on my side, so this may indeed be the last change from my side that would not contribute some content. Happy!

We will see ;-)

**Review by hansu — APPROVED** [review]

**Pull request merged**